sopwith3.txt

```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Maelstorm
+Version = 3.0.6
+License = GPL
+Description = Maelstrom is a GPL'd port of the original shareware game for the Macintosh. It is a fast-action, high resolution (640x480) asteroids-like game, with detailed graphics and original sounds.
```

Comment:
Name and description is copypasted from [maelstorm.txt](https://github.com/reactos/rapps-db/pull/174/files#diff-260fdf3789fa437d8969d03106e16086395922081da45aeda8470434d6ea40fe), but this is definitely something different.